docker-compose-files/dspace-oracle-compose/README.md

```diff
@@ -49,24 +49,24 @@ The dspace container and the dspacedb container will persist data in a docker vo
 
 #### Bash
 ```
-docker exec -it ${DPROJ}_dspace_1 /dspace/bin/dspace version
+docker exec -it dspace /dspace/bin/dspace version
```

Comment:
This is really neat and useful! :+1:

Reply:
I will ask about this on the DEV channel to understand how these values can be provided to a build.

Reply:
https://github.com/DSpace-Labs/DSpace-Docker-Images/issues/72 created to figure out a mechanism for this.  It will likely require passing a variable from the dockerhub build system.